backend/src/controllers/person.controller.ts

```diff
@@ -1,31 +1,21 @@
 /**
  * Controller contains high-level operations using services, consumed by routes
  */
-import { NextFunction, Request, Response } from 'express';
-import { createPerson as createPersonService } from '../services/person.service';
+import { Request, Response } from 'express';
+import httpStatus from 'http-status';
+import personService from '../services/person.service';
+import logger from '../utils/logger';
 
-export const getRandomJson = async (
-  req: Request,
-  res: Response,
-  next: NextFunction,
-): Promise<void> => {
-  try {
-    res.json({ someString: 'hello world' });
-  } catch (e) {
-    next(e);
-  }
-};
+import { POST } from './controller.types';
+
+export const createPerson: POST = async (req: Request, res: Response): Promise<void> => {
+  logger.info("POST /persons/create request from frontend");
 
-export const createPerson = async (
-  req: Request,
-  res: Response,
-  next: NextFunction,
-): Promise<void> => {
-  const { name, organisation } = req.body;
   try {
-    await createPersonService({ name, organisation });
-    res.status(200).end();
+    const person = await personService.createPerson(req.body);
+    res.status(httpStatus.CREATED).send(person);
   } catch (e) {
-    next(e);
+    logger.error(e);
+    res.status(httpStatus.BAD_REQUEST);
```

Comment:
Was there a particular reason you avoided using next(e)? My rationale behind using next(e) is that you can then pass error handling to the errorHandler defined in `backend/src/utils/middleware`. This way, you don't need to do logger.error(e) and res.status() every time you handle error

Reply:
I actually completely missed this, I think if I just replace the current `400` value in the errorHandler to `httpStatus.BAD_REQUEST` it should work the same 👍